scss/_login.scss

```diff
@@ -0,0 +1,53 @@
+.container {
+  justify-content: center;
+}
+.login {
+  @include light-blue-border;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  max-width: 400px;
+  min-height: 550px;
```

Comment:
Better use a max-heght and height to set this.

Reply:
Done 😄